plugins/woocommerce/includes/class-wc-form-handler.php

```diff
@@ -132,7 +132,7 @@ public static function save_address() {
 			// Validation: Required fields.
 			if ( ! empty( $field['required'] ) && empty( $value ) ) {
 				/* translators: %s: Field name. */
-				wc_add_notice( sprintf( __( '%s is a required field.', 'woocommerce' ), $field['label'] ), 'error', array( 'id' => $key ) );
```

Comment:
We had a mixture of `%s is a required field` and `%s is a required field.` so I standardised on one.